episodes/13-synchronization.md

```diff
@@ -210,19 +207,16 @@ x.readFF()		//will block until the state of x is full,
 //non-blocking read and write methods
 x.writeXF(value)	//will assign the value no matter the state of x, and then set the state as full
```

Comment:
What do `XF` and `XX` mean?

Reply:
"X" means any state. So, "XF" means go from any state (empty or full) to full, hence it is non-blocking. "XX" means go from any state (empty or full), read the variable, but leave the state unchanged (empty or full). I don't think I need to explain this, as the comments explain that already.